### DISCUSSION TITLE
Keybinds shortcuts

### DISCUSSION THREAD

**Discussion by CookiesHavePoison** [rocksdanister/lively#1515] [Ideas]
Maybe options to set keybinds to:
- Start / pause wallpaper playback
- Change taskbar theme
- Remove taskbar theme
- Change wallpaper(s)
- Remove wallpaper(s)
- Mute / unmute wallpaper audio
- Change volume of wallpaper audio

It's a niche feature but yea. Also, ^ is not an exaustive list, just what I had off the top of my head.

**Comment by rocksdanister** [comment]
You can use AutoHotkey and Lively commandline controls to achieve this, some examples are provided here:
https://github.com/rocksdanister/lively/wiki/Command-Line-Controls

**Reply by CookiesHavePoison** [reply]
Thanks for the response! 

May I ask if there are plans to bundle the command utility with the actual application installation (for MS version)? I believe that to implement a keybind feature natively to the app would hugely improve ease of access and quality of life. Using AutoHotkey and writing scripts for it requires learning how to program (in a new language). A first-party implementation of keybinding and scripting (with a GUI) is much more userfriendly and would encourage users to push the boundaries of what is possible with the app. 

I appreciate that there are probably more important things for you to implement / fix and I guess I'm just lazy but I do feel that it would be helpful. Besides, it's another feature to pitch the app with, especially since as far as I am aware, Wallpaper Engine also does not natively support keybinds - only command line controls. 

I apologise if I have wasted your time here and hope that you have a good day.

**Reply by rocksdanister** [reply]
>May I ask if there are plans to bundle the command utility with the actual application installation (for MS version)?

I don't understand what you mean by this? why bundle a commandline interface when there is already a gui interface?

>Using AutoHotkey and writing scripts for it requires learning how to program (in a new language).

You asked for keybind, the ahk example in the wiki is pretty straightforward and most of the programming is for the commandline utility arguments.

>A first-party implementation of keybinding and scripting (with a GUI) is much more userfriendly and would encourage users to push the boundaries of what is possible with the app.

Implementing this is not hard but like you said its a niche feature and not a priority at the moment; this is what commandline utility is meant for.
